201-vm-vsts-agent/azuredeploy.json

```diff
@@ -74,7 +67,21 @@
             "metadata": {
                 "description": "The Visual Studio Team Services build agent pool for this build agent to join. Use 'Default' if you don't have a separate pool."
             }
-        }        
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-vsts-agent/scripts/InstallVstsAgent.ps1"
```

Comment:
this is just the location for the folder/container for the file, not the filename itself - so here will be: 
https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-vsts-agent

And then use a var or concat inline for the /scripts/InstallVstsAgent.ps1 part of the path